llvm/lib/Bitcode/Reader/MetadataLoader.cpp

```diff
@@ -1702,10 +1703,13 @@ Error MetadataLoader::MetadataLoaderImpl::parseOneMetadata(
       if (Record.size() > 23) {
         Specification = getMDOrNull(Record[23]);
       }
+      // FIXME
+      if (Record.size() > 25)
+        BitStride = getMDOrNull(Record[25]);
     }
 
-    if (Record.size() > 25 && Record[25] != dwarf::DW_APPLE_ENUM_KIND_invalid)
-      EnumKind = Record[25];
+    if (Record.size() > 24 && Record[24] != dwarf::DW_APPLE_ENUM_KIND_invalid)
+      EnumKind = Record[24];
```

Comment:
FYI, fixed separately by https://github.com/llvm/llvm-project/pull/132374

Reply:
If that lands first I will rebase onto it.